test/edu/ucsb/nceas/metacat/dataone/D1AuthHelperTest.java

```diff
@@ -81,33 +106,44 @@ public static void setUpBeforeClass() {
         nl.addNode(otherMN);
     }
 
-    D1AuthHelper authDel;
-    D1AuthHelper authDelMock;
-    Session session;
-    Session authMNSession;
-    Session otherMNSession;
-    Session replMNSession;
-    Session cn1CNSession;
-    SystemMetadata sysmeta;
-
     /**
      * Get a minimal SystemMetadata object with default values
      */
     private SystemMetadata getGenericSysmetaObject() throws Exception {
-        SystemMetadata sysmeta =
-            TypeFactory.buildMinimalSystemMetadata(TypeFactory.buildIdentifier("dip"),
-                                                   new ByteArrayInputStream(
-                                                       ("tra la la la la").getBytes("UTF-8")),
-                                                   "MD5",
-                                                   TypeFactory.buildFormatIdentifier("text/csv"),
-                                                   TypeFactory.buildSubject(
-                                                       "submitterRightsHolder"));
+        SystemMetadata sysmeta = TypeFactory.buildMinimalSystemMetadata(
+            TypeFactory.buildIdentifier("dip"), new ByteArrayInputStream(
+                ("Test Sysmeta Content InputStream").getBytes(StandardCharsets.UTF_8)), "MD5",
+            TypeFactory.buildFormatIdentifier("text/csv"),
+            TypeFactory.buildSubject("submitterRightsHolder"));
         AccessPolicy ap = new AccessPolicy();
         ap.addAllow(TypeFactory.buildAccessRule("eq1", Permission.CHANGE_PERMISSION));
         sysmeta.setAccessPolicy(ap);
         return sysmeta;
     }
 
+    /**
+     * Get a rightsHolderGroup, if null is supplied default values will be used.
+     */
+    private List<Group> getGenericRightsHolderGroup(List<Subject> subjectMemberList) {
```

Comment:
I've see a general convention of putting `private` methods at the bottom of the class

Reply:
Noted, I'll keep this in mind moving forward. Thank you!